content/zh/blog/2018/egress-tcp/index.md

```diff
@@ -339,7 +339,7 @@ bypassing the sidecar proxies.
 ## Relation to mesh expansion
 
 Note that the scenario described in this post is different from the mesh expansion scenario, described in the
-[Bookinfo with Mesh Expansion](/docs/examples/mesh-expansion/bookinfo-expanded/) example. In that scenario, a MySQL instance runs on an
+[Bookinfo with Mesh Expansion](/docs/examples/virtual-machines/bookinfo) example. In that scenario, a MySQL instance runs on an
```

Comment:
Mesh Expansion

Reply:
Sorry, but these are blog posts in the ZH side of things, I don't want to update the English basis that folks are using for translation. I just fixed the link for the linter.